apps/docs/demos/alert/banner-usage/App.tsx

```diff
@@ -7,12 +7,7 @@ export function App() {
   return (
     <>
       {open && (
-        <Alert
-          colorScheme="danger"
-          onClose={() => setOpen(false)}
-          variant="solid"
-          w="full"
-        >
+        <Alert colorScheme="danger" onClose={() => setOpen(false)} w="full">
```

Comment:
I think we wont need banner usage anymore as we are exposing `banner` component individually.

Reply:
let's just remove this altogether in that case

Reply:
Yes, I meant that